src/Sentry/SentryHttpFailedRequestHandler.cs

```diff
@@ -15,67 +15,84 @@ internal SentryHttpFailedRequestHandler(IHub hub, SentryOptions options)
     protected internal override void DoEnsureSuccessfulResponse([NotNull] HttpRequestMessage request, [NotNull] HttpResponseMessage response)
```

Comment:
suggestion: update from `main`, as we've merged the big `version6` branch adding support for .NET 10 in the meantime